src/main/java/com/palantir/gradle/versions/GradleWorkarounds.java

```diff
@@ -169,9 +165,10 @@ private static boolean isPlatformPre53(AttributeContainer attributes) {
     }
 
     static boolean isFailOnVersionConflict(Configuration conf) {
-        org.gradle.api.internal.artifacts.configurations.ConflictResolution conflictResolution =
-                ((org.gradle.api.internal.artifacts.configurations.ResolutionStrategyInternal) conf
-                        .getResolutionStrategy()).getConflictResolution();
+        org.gradle.api.internal.artifacts.configurations.ConflictResolution conflictResolution = ((org.gradle.api
+                                .internal.artifacts.configurations.ResolutionStrategyInternal)
```

Comment:
wooooah not sure about this one.  breaking midway through a fully-qualified name seems not nice

Reply:
I think this could be fixedby trying both "break initializer" (current behaviour) and newline after `=`.

Currently it will only allow a newline after `=` if the whole initializer fits on one line.